try {

} catch (e) {
    let i = 0
}
